src/NzbDrone.Core/Organizer/FileNameBuilder.cs

```diff
@@ -122,6 +122,7 @@ public string BuildTrackFileName(List<Track> tracks, Artist artist, Album album,
             AddAlbumTokens(tokenHandlers, album);
             AddMediumTokens(tokenHandlers, tracks.First().AlbumRelease.Value.Media.SingleOrDefault(m => m.Number == tracks.First().MediumNumber));
             AddTrackTokens(tokenHandlers, tracks);
+            AddTrackArtistTokens(tokenHandlers, tracks);
```

Comment:
This logic can be added to `AddTrackTokens` instead of making a new module given this is track metadata